stream_alert/rule_processor/classifier.py

```diff
@@ -166,13 +167,14 @@ def _check_schema_match(schema_matches):
         matches = []
         for i, schema_match in enumerate(schema_matches):
             log_patterns = schema_match.parser.options.get('log_patterns', {})
+            LOGGER.debug('Log patterns: %s', log_patterns)
             if (all(schema_match.parser.matched_log_pattern(data, log_patterns)
                     for data in schema_match.parsed_data)):
                 matches.append(schema_matches[i])
             else:
                 LOGGER.debug(
-                    'Log pattern matching failed for schema: %s',
-                    schema_match.root_schema)
+                    'Log pattern matching failed for: \n%s',
+                    json.dumps(schema_match.parsed_data, indent=4))
```

Comment:
can we stick to using `indent=2` for all json dumps calls? also huge nit, but unnecessary space between `:` and `\n` above